src/System.Private.ServiceModel/tests/Common/Infrastructure/xunit/WcfFactDiscoverer.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Infrastructure.Common
+{
+    public class WcfFactDiscoverer : FactDiscoverer
```

Comment:
Maybe add comment referencing [https://github.com/dotnet/buildtools/blob/master/src/xunit.netcore.extensions/Discoverers/ConditionalFactDiscoverer.cs](https://github.com/dotnet/buildtools/blob/master/src/xunit.netcore.extensions/Discoverers/ConditionalFactDiscoverer.cs) as inspiration for this in case things get refactored, it will be easier to know where to look to see what changes we need to make.